.bazelrc

```diff
@@ -23,3 +23,5 @@ build:android --fat_apk_cpu=x86,x86_64
 build:device --ios_multi_cpus=armv7,arm64 --fat_apk_cpu=armeabi-v7a,arm64-v8a
 
 build:fat --ios_multi_cpus=i386,x86_64,armv7,arm64 --fat_apk_cpu=x86,x86_64,armeabi-v7a,arm64-v8a
+
+build:sizeopt -c opt --copt -Os --copt=-ggdb3 --linkopt=-fuse-ld=lld
```

Comment:
I don't think lld would be compatible with the iOS build?

Reply:
Can this also just get rolled into the release PR that @rebello95 is working on instead?

Reply:
@keith does iOS use gold?

I can work with @rebello95 to consolidate. I do think that for release we want to do `--strip=never`, while here we want to keep symbols and manually strip after bazel.

Reply:
Apple has their own linker, ld64, and we have to use that one

Reply:
I see, thanks for the info. My idea with the sizeopt config was to use to get binaries for size analysis, not necessarily for release. So the linker stipulation here is just for standardization of the tooling used in that type of investigation.

I'll adjust the bazel config accordingly once we settle on the release config.

Reply:
This would likely just mean that we wouldn't be able to use `--config=sizeopt` with iOS, right now @rebello95's open PR is using that config in the release mode

Reply:
gotcha, ok. I just looked at @rebello95's PR. I think I was confused about the ordering here. Now I see that #155 depends on this. I have fixed in a way that I think makes everything work together lmk what you think @keith